database/migrations/2020_01_08_043027_create_blogs_table.php

```diff
@@ -0,0 +1,34 @@
+<?php
+
+use Illuminate\Database\Migrations\Migration;
+use Illuminate\Database\Schema\Blueprint;
+use Illuminate\Support\Facades\Schema;
+
+class CreateBlogsTable extends Migration
+{
+    /**
+     * Run the migrations.
+     *
+     */
+    public function up()
+    {
+        Schema::create('blogs', function (Blueprint $table) {
+            $table->bigIncrements('id');
```

Comment:
where is the usage of https://docs.google.com/spreadsheets/d/1bqeqmlFx-0867eX4onaM6IZYLpkewONk3wI8i1vICTs/edit#gid=41579321 Creating Indexes

Reply:
Em thêm các cột mới: user_id, title, content, image_urls, deleted_at và dùng các cột có sẵn là: id và created_at, updated_at